task.sql

```diff
@@ -1,17 +1,37 @@
 CREATE DATABASE ShopDB; 
 USE ShopDB; 
 
--- Create a table to store countries 
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 ) ENGINE=InnoDB;
 
--- Create a table for caching GeoIP data (Columns: ID, IP Range, CountryID)
+CREATE TABLE GeoIPCache (
+    ID INT AUTO_INCREMENT,
+    IPRange VARCHAR(100),
```

Comment:
The `GeoIPCache` table uses the `Memory` storage engine, which does not support `TEXT` or `BLOB` data types. Consider changing the `IPRange` column to a supported data type like `VARCHAR` with a length that fits your requirements.